fem/linearform.hpp

```diff
@@ -97,15 +104,22 @@ class LinearForm : public Vector
    /** The associated FiniteElementSpace can be set later using one of the
        methods: Update(FiniteElementSpace *) or
        Update(FiniteElementSpace *, Vector &, int). */
-   LinearForm() { fes = NULL; ext = nullptr; extern_lfs = 0; UseDevice(true); }
+   LinearForm()
+   {
+      fes = NULL; ext = nullptr; extern_lfs = 0; UseDevice(true);
```

Comment:
We could have the other constructors call this default constructor to set these default values and reduce duplication.